packages/ponytest/ponytest.pony

```diff
@@ -223,6 +244,11 @@ actor PonyTest
       return
     end
 
+    // Ignore tests when label arg is set and test label doesn't match
+    if (_label!="") and (_label!=test.label()) then
```

Comment:
small formatting thing:
there should be 

`_label != ""` 

and

`_label != test.label()